addons/web_editor/static/src/js/widgets.js

```diff
@@ -879,57 +873,104 @@ var VideoDialog = Widget.extend({
     },
     start: function () {
         this.$preview = this.$('.preview-container').detach();
-        this.$iframe = this.$("iframe");
+        this.$content = this.$(".o_video_dialog_iframe");
         var $media = $(this.media);
+
         if ($media.hasClass("media_iframe_video")) {
             var src = $media.data('src');
-            this.$("input#urlvideo").val(src);
-            this.$("input#autoplay").prop("checked", (src || "").indexOf("autoplay") >= 0);
-            this.get_video();
+            this.$("textarea#o_video_text").val(src);
+
+            this.$("input#o_video_autoplay").prop("checked", (src || "").indexOf("autoplay=1") >= 0);
+            this.$("input#o_video_hide_controls").prop("checked", (src || "").indexOf("controls=0") >= 0);
+            this.$("input#o_video_loop").prop("checked", (src || "").indexOf("loop=1") >= 0);
+            this.$("input#o_video_hide_fullscreen").prop("checked", (src || "").indexOf("fs=0") >= 0);
+            this.$("input#o_video_hide_yt_logo").prop("checked", (src || "").indexOf("modestbranding=1") >= 0);
+            this.$("input#o_video_hide_dm_logo").prop("checked", (src || "").indexOf("ui-logo=0") >= 0);
+            this.$("input#o_video_hide_dm_share").prop("checked", (src || "").indexOf("sharing-enable=0") >= 0);
+
+            this._getData(event);
         }
         return this._super.apply(this, arguments);
     },
-    change_input: function (e) {
-        var $input = $(e.target);
-        var $button = $input.parent().find("button");
-        if ($input.val() === "") {
-            $button.addClass("btn-default").removeClass("btn-primary");
-        } else {
-            $button.removeClass("btn-default").addClass("btn-primary");
-        }
+    _reset_feedback: function (e) {
+        this.$content.empty();
+        this.$el.find('#o_video_form_group').removeClass('has-error has-success');
+    },
+    _userWriting : function () {
+        var self = this;
+        this._reset_feedback();
+        _.throttle(self._getData(),1000);
```

Comment:
`_.throttle` expects a function, not a function call. So `_.throttle(self._getData(),1000);` is exactly the same as `self._getData()`. I think that what you want is to do `this._userWriting = _.debounce(this._userWriting, 1000);` in the `init` method but I am not sure...